codalab/apps/web/static/js/worksheet/worksheet_interface.js

```diff
@@ -148,28 +152,26 @@ var Worksheet = React.createClass({
           $("#raw-textarea").focus();
         }
     },
-    toggleActionBar: function(){
-        this.setState({showActionBar:!this.state.showActionBar});
+    toggleActionBar: function() {
+        this.setState({showActionBar: !this.state.showActionBar});
     },
-    showActionBar: function(){
-        this.setState({showActionBar:true});
+    showActionBar: function() {
+        this.setState({showActionBar: true});
     },
-    hideActionBar: function(){
-        this.setState({showActionBar:false});
+    hideActionBar: function() {
+        this.setState({showActionBar: false});
     },
-    refreshWorksheet: function(){
+    refreshWorksheet: function() {
         $('#update_progress').show();
-        this.setState({updating:true});
+        this.setState({updating: true});
         ws_obj.fetch({
-            success: function(data){
-                // console.log("fetch_and_update success");
-                // console.log("%c--------------------------------------------------------------", "color: Green; font-size:15px;");
-                if(this.isMounted()){
+            success: function(data) {
+                if (this.isMounted()) {
                     this.refs.list.setState({worksheet: ws_obj.getState()});
```

Comment:
The changes look good. :+1: 

One comment on the React way of doing things though. Technically, since React creates a virtual DOM to figure out which part of the actual DOM need to be updated, all the JQuery code should not be a part of our component class. Components should just be accessing the data passed in the parent in `this.props` and changing its state by modifying `this.state`. For all the JQuery commands, we should ideally just have the logic and the variables checking things need to be shown or hidden and when the data would change, the logic described by the components render and ComponentDidMount method would help create the virtualDOM that would update the div state itself. But that would require major restructuring of code and creation of more components that handle the error and other JQuery logic.